src/dagmc/DagMC.hpp

```diff
@@ -212,44 +212,231 @@ class DagMC {
+   * closest point lies. Default is 0.
+   *
+   * @return Returns an ErrorCode indicating the success or failure of the
+   * operation.
+   */
   ErrorCode closest_to_location(EntityHandle volume, const double point[3],
                                 double& result, EntityHandle* surface = 0);
 
+  /**
+   * @brief Measures the volume of a specified volume.
+   *
+   * @param volume The volume to be measured.
+   * @param[out] result The measured volume.
+   *
+   * @return Returns an ErrorCode indicating the success or failure of the
+   * operation.
+   */
   ErrorCode measure_volume(EntityHandle volume, double& result);
 
+  /**
+   * @brief Measures the area of a specified surface.
+   *
+   * @param surface The surface to be measured.
+   * @param[out] result The measured area.
+   *
+   * @return Returns an ErrorCode indicating the success or failure of the
+   * operation.
+   */
   ErrorCode measure_area(EntityHandle surface, double& result);
 
+  /**
+   * @brief Determines the sense of one or more surfaces with respect to a
+   * specified volume.
+   *
+   * This method assumes that the surfaces passed in are part of the volume.
+   *
+   * @param volume The volume with respect to which the sense is determined.
+   * @param num_surfaces The number of surfaces for which to determine the
+   * sense.
+   * @param surfaces An array of handles of the surfaces for which to determine
+   * the sense.
+   * @param[out] senses_out An array in which to store the determined senses.
+   *
+   * @return Returns an ErrorCode indicating the success or failure of the
+   * operation.
+   */
   ErrorCode surface_sense(EntityHandle volume, int num_surfaces,
                           const EntityHandle* surfaces, int* senses_out);
 
+  /**
+   * @brief Determines the sense of a surface with respect to a specified
+   * volume.
+   *
+   * This method assumes that the surface passed in is part of the volume.
+   *
+   * @param volume The volume with respect to which the sense is determined.
+   * @param surface The handle of the surface for which to determine the sense.
+   * @param[out] sense_out The determined sense.
+   *
+   * @return Returns an ErrorCode indicating the success or failure of the
+   * operation.
+   */
   ErrorCode surface_sense(EntityHandle volume, EntityHandle surface,
                           int& sense_out);
 
+  /**
+   * @brief Returns the normal vector of a surface at the specified point. from
+   * the ray origin in a specified direction. It is assumed that the specified
+   * point is on the surface.
+   *
+   * This method first identifies which triangle contains this point and then
+   * calculates the unit outward normal of that triangle.  The triangle of the
+   * provided volume that is nearest the provided point is used for this
+   * calculation. The search for that triangle can be circumvented by providing a
+   * RayHistory, in which case the last triangle of the history will be used.
```

Comment:
It looks like our `clang-format` wants to wrap this line differently 🤷 

```suggestion
   * calculation. The search for that triangle can be circumvented by providing
   * a RayHistory, in which case the last triangle of the history will be used.
```

Reply:
No worries! Ran clang-format locally and pushed before I saw this. Sorry to not incorporate the suggestion!